source/02_algo/exploration_rl_en.rst

```diff
@@ -0,0 +1,241 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Intrinsic reward based on prediction error is  \ **to use the prediction error of a state on a prediction problem (usually a supervised learning problem) to measure novelty**\. According to the characteristics of neural network fitting datasets in supervised learning, if the agent has a larger prediction error in a certain state \ :math:`s`\, the approximate state space in the vicinity of the state \ :math:`s`\. The number of previous visits by the upper agent is small, so the state\ :math:`s`\ is more novel.
+
+Prediction problems are often problems related to the dynamics of the environment, such as the paper [3]_ `Curiosity-driven Exploration by Self-supervised Prediction <http://proceedings.mlr.press/v70/pathak17a/pathak17a.pdf>`__ (ICM) proposed a new Intrinsic Curiosity module based on prediction error. Module, ICM), by using the inverse dynamics model and the forward dynamics model to learn a new feature space on the original problem space, so that the learned feature space only encodes the part that affects the agent's decision-making, while ignoring the environment noise and other irrelevant interference. Then on this purer feature space, the prediction error of the forward model is used to provide the intrinsic for RL training reward. For more details about ICM, please refer to \ `blog <https://zhuanlan.zhihu.com/p/473676311>`__\ .
+
+But ICM has the following problems:
+
+-  On large-scale problems, the complex forward dynamics model of the environment, coupled with the limited capacity of the neural network, leads to large prediction errors when certain regions of the state-action space are visited a large number of times.
+
+-  In some environments, the state transition function of the environment is a random function, such as an environment containing noise-TV properties, and the agent cannot accurately predict the next state through the usual neural network.
+
+In order to alleviate the above problems, the paper [4]_ `Exploration by Random Network
+Distillation <https://arxiv.org/abs/1810.12894v1>`__\  proposes the RND algorithm, which is also an exploration method based on the prediction problem, but special is that the prediction in the RND algorithm The problem is a stochastic distillation problem related only to the observation state, not a forward or inverse dynamical model of the environment. Specifically, RND utilizes two neural networks with the same structure: a target network with fixed random initialization parameters. A predictor network network), the predictor network is used to output the predicted value of the state encoding given to the target network. Then the RND intrinsic exploration reward is defined as being proportional to the state feature predicted by the predictor network\ :math:`\hat{f}( s_t )`\ and the state feature of the target network :math:`f(s_t)`\  . For more details about RND, please refer to \ `blog <https://zhuanlan.zhihu.com/p/473676311>`__\.
```

Comment:
Grammar mistake: "but special is that the prediction in the RND algorithm" and "a target network with fixed random initialization parameters. A predictor network network)"